src/views/clusteroverview/OverviewTab/resources-inventory-card/ResourcesInventoryCard.tsx

```diff
@@ -0,0 +1,75 @@
+import React from 'react';
+
+import { NodeModel } from '@kubevirt-ui/kubevirt-api/console';
+import NetworkAttachmentDefinitionModel from '@kubevirt-ui/kubevirt-api/console/models/NetworkAttachmentDefinitionModel';
+import VirtualMachineModel from '@kubevirt-ui/kubevirt-api/console/models/VirtualMachineModel';
+import { ALL_NAMESPACES_SESSION_KEY } from '@kubevirt-utils/hooks/constants';
+import { useIsAdmin } from '@kubevirt-utils/hooks/useIsAdmin';
+import { useKubevirtTranslation } from '@kubevirt-utils/hooks/useKubevirtTranslation';
+import { getResourceUrl } from '@kubevirt-utils/resources/shared';
+import { useActiveNamespace } from '@openshift-console/dynamic-plugin-sdk-internal';
+import { Card, Grid, GridItem } from '@patternfly/react-core';
+
+import useResourcesQuantities from './hooks/useResourcesQuantities';
+import ResourceInventoryItem from './ResourceInventoryItem';
+
+import './ResourcesInventoryCard.scss';
+
+const ResourcesInventoryCard: React.FC = () => {
+  const { t } = useKubevirtTranslation();
+  const isAdmin = useIsAdmin();
+
+  const [activeNamespace] = useActiveNamespace();
+  const namespace = React.useMemo(
+    () => (activeNamespace === ALL_NAMESPACES_SESSION_KEY ? null : activeNamespace),
+    [activeNamespace],
+  );
+  const { networks, nodes, vms, vmTemplates } = useResourcesQuantities(namespace);
```

Comment:
im guessing this hook can also consume useactivename space hook and no need to transfer namespace

Reply:
Good point. Somehow, I didn't notice that I wasn't using `namespace` anywhere else in this file. Thanks for catching it!